deploy-as-code/helm/environments/works-dev.yaml

```diff
@@ -175,6 +175,9 @@ cluster-configs:
 estimate-service:
   tracing-enabled: true
   java-enable-debug: true
+  estimate-default-limit: 100
+  estimate-default-offset: 0

```

Comment:
Do you really want to allow this to be configurable?
You can keep this as part of application.properties. 
Do other projects have it here?

Reply:
Yes. If you remember, we had a discussion in IFIX about limits. If we'll keep them as part of environment variables. we can override them in the respective environment.  I checked in PGR service, those are not there. Let me know your thought.

Reply:
It makes sense to keep limits. But doesn't having offset anything other than 0 create problems?

Reply:
ok, I'll remove the offset environment variable from the env file and corresponding chart file.